tedana/resources/config/metrics.json

```diff
@@ -13,10 +13,18 @@
             "map FT2",
             "map Z"
         ],
+        "kappa rank": [
+            "map FT2",
+            "map Z"
+        ],
```

Comment:
Shouldn't this just be calculated from `kappa`?

Reply:
It should. But since `kappa rank` is the same data but ranked I thought it should contain the same attributes.

Reply:
I think we should just consider basic "rank" versions of metrics as derivatives of just the metrics they're ranking. The dependency graph will have more layers (i.e., `kappa rank` --> `kappa` --> `map Z` + `map FT2` instead of `kappa rank` --> `map Z` + `map FT2`), but it should be more interpretable that way.

Reply:
I agree with this. I like the idea of using the attributes to more clearly show which ones are derivative. Perhaps it's obvious that kappa_rank is just kappa values ranked, setting up this rule now might make future additions of metrics a bit more clear. This is already what we do with d_table_score.